lib/meta-boxes.php

```diff
@@ -119,6 +126,20 @@ function igv_cmb_metaboxes() {
   ) );
 
   $obra_meta->add_field( array(
+    'name'       => __( 'Image', 'cmb2' ),
+    'desc'       => __( 'if single image not slideshow', 'cmb2' ),
+    'id'         => $prefix . 'image',
+    'type'       => 'file',
+  ) );
+
+  $obra_meta->add_field( array(
+    'name'       => __( 'Video', 'cmb2' ),
+    'desc'       => __( 'if video not slideshow. [vimeo embed id or ???]', 'cmb2' ),
```

Comment:
yh tbh i need to email about this feature. wasnt ever in any of the pdfs but obv a vimeo/youtube embed is very easy. I prefer IDs to urls and oembed tbh but isnt that different. I guess leave this as placeholder and do a video feature branch when we kno